.gitlab-ci.yml

```diff
@@ -33,7 +33,7 @@ Build stage Docker image:
     - docker tag $IMAGE_NAME:$CI_BUILD_REF $DOCKER_REPO_INFRA_STAGE:$CI_BUILD_REF
     - $DOCKER_LOGIN_TO_INFRA_STAGE_REPO && docker push $DOCKER_REPO_INFRA_STAGE:$CI_BUILD_REF
   only:
-    - stage
+    - fix/panic-in-ProcessSyncMessage
```

Comment:
lets change back to stage

Reply:
@nivBlox Sure, I kept it for a case if I still can find the panic in Kibana logs (to add more code). I don't see any logs in Kibana that could trigger this panic, so it seems that this panic is not reproduced in our deployment and panics in Kibana are caused by some other part of code. Therefore, I think we can remove debug logs now.